x-pack/plugins/lens/public/metric_visualization/visualization.tsx

```diff
@@ -56,22 +78,87 @@ const toExpression = (
     reverse: false,
   };
 
+  const fontSizes: Record<string, { size: number; sizeUnit: string }> = {
```

Comment:
It seems like this mapping is pretty different than the old one: https://github.com/elastic/kibana/pull/126019/files#diff-2c977ca1fa0b45f15d61f5c8722ef7209c1e01f521a69761454599b074203778L37-L85

I don't have a super strong opinion on this but especially the small sizes feel too large with this new setting - it's very useful to create small metrics, e.g. for solutions (left is new, right is old).
<img width="1729" alt="Screenshot 2022-03-01 at 18 17 26" src="https://user-images.githubusercontent.com/1508364/156216767-4ccba093-c8c0-4c55-b91d-76f371f5deb2.png">

I would suggest going with the old mapping

Reply:
@flash1293 I tried to find a general approach so that the ratio of the size of the metric to the size of the labels is always the same, because before each metric size was chosen according to different rules depending on the size of the label. You can see it here: https://github.com/elastic/kibana/pull/126019/files#diff-2c977ca1fa0b45f15d61f5c8722ef7209c1e01f521a69761454599b074203778L37-L85

If you would like I can back to that solution and update mapping

Reply:
I promised to have a look here and I just wanted to confirm that Joe's solution will be ok from my point of view!

Reply:
@flash1293 I updated font size mapping